pyEpiabm/pyEpiabm/intervention/place_closure.py

```diff
@@ -6,9 +6,10 @@
 
 
 class PlaceClosure(AbstractIntervention):
-    """Place closure intervention
-    Close places based on the number of icu patients and infecious persons
-    in their microcells and reopen places after their closure period.
+    """Place closure intervention.
+    Close places based on the number of ICU patients and infectious persons
+    in their microcells and reopen places after their closure period or
+    after the end of the policy.
```

Comment:
Expand docstring to make it clear whether you need to be over both the ICU patient and the infectious person threshold, or just one of the these. And do you need to be past the start time as well? How do these combine?

Reply:
This is solved on another branch (social_distancing) but have changed it here now as well.